Source/Engine/Physics/Actors/RigidBody.cpp

```diff
@@ -327,12 +327,10 @@ void RigidBody::OnColliderChanged(Collider* c)
 
 void RigidBody::UpdateBounds()
 {
-    void* actor = _actor;
-    if (actor && PhysicsBackend::GetRigidActorShapesCount(actor) != 0)
-        PhysicsBackend::GetActorBounds(actor, _box);
-    else
-        _box = BoundingBox(_transform.Translation);
-    BoundingSphere::FromBox(_box, _sphere);
+    const float radius = GetScale().MaxValue();
```

Comment:
there i a need to sync the BB with PhysicsBackend first

Reply:
But PhysicsBackend::GetRigidActorShapesCount is making this problem, before CharacterController had this problem with BB too and got fixed removing this and adding this code: https://github.com/FlaxEngine/FlaxEngine/blob/9da686cd525c3ff76fc5b8722b697d26c64688a1/Source/Engine/Physics/Colliders/CharacterController.cpp#L244C32-L244C32

My changes is a "copy" of the CharacterController BB to fix this problem